file-compiler-plugin-test/src/test/java/io/ballerina/stdlib/file/compiler/CompilerPluginTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.file.compiler;
+
+import io.ballerina.projects.DiagnosticResult;
+import io.ballerina.projects.Package;
+import io.ballerina.projects.PackageCompilation;
+import io.ballerina.projects.ProjectEnvironmentBuilder;
+import io.ballerina.projects.directory.BuildProject;
+import io.ballerina.projects.environment.Environment;
+import io.ballerina.projects.environment.EnvironmentBuilder;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * File compiler plugin tests.
+ */
+public class CompilerPluginTest {
+
+    private static final Path RESOURCE_DIRECTORY = Paths.get("src", "test", "resources", "test-src")
+            .toAbsolutePath();
+    private static final Path DISTRIBUTION_PATH = Paths.get("build", "target", "ballerina-distribution")
+            .toAbsolutePath();
+
+    @Test
+    public void testCompilerPlugin() {
+        Package currentPackage = loadPackage("package_01");
+        PackageCompilation compilation = currentPackage.getCompilation();
+
+        DiagnosticResult diagnosticResult = compilation.diagnosticResult();
+        Assert.assertEquals(diagnosticResult.diagnostics().size(), 0);
+    }
+
+    @Test
+    public void testCompilerPluginWithInvalidParams() {
+        Package currentPackage = loadPackage("package_02");
+        PackageCompilation compilation = currentPackage.getCompilation();
+        String errMsg = "ERROR [file_service.bal:(29:4,31:5)] The remote function should contain a parameter";
+        String errMsg1 = "ERROR [file_service.bal:(33:4,35:5)] Invalid function name `onEdit`";
+        String errMsg2 = "ERROR [file_service.bal:(37:4,39:5)] The remote function should contain a parameter";
```

Comment:
Shall we mention which parameter is missing here. Something like `The remote function should only contain `file:FileEvent ` parameter`

Reply:
Shall we make all error messages start in lowercase align with compiler error messages which is lower cases